src/tools/tidy/src/issues.txt

```diff
@@ -853,6 +853,8 @@ ui/coroutine/issue-87142.rs
 ui/coroutine/issue-88653.rs
 ui/coroutine/issue-91477.rs
 ui/coroutine/issue-93161.rs
+ui/cross-crate/issue-38875/auxiliary/issue-38875-b.rs
+ui/cross-crate/issue-38875/issue-38875.rs
```

Comment:
The original issue was named ["constant evaluation error" points to nothing when crossing the crate boundary](https://github.com/rust-lang/rust/issues/38875), the solution PR was [implement lazy-loading of crate's sources](https://github.com/rust-lang/rust/pull/42593), could you rename this for that? Perhaps the root one should be
- tests/ui/cross-crate/lazy-load-externs/const-issue-38875.rs